addons/mail/static/src/components/composer_text_input/composer_text_input.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ComposerTextInput" owl="1">
-        <div class="o_ComposerTextInput">
+        <div class="o_ComposerTextInput" t-on-paste="props.onPaste">
```

Comment:
Where is that coming from? It doesn't look like it follows the "procedure" done in the rest of the diff.

Reply:
There was a `<ComposerTextInput t-on-paste>` in `composer.xml`.